### PR TITLE
78 artist profile

### PR DESCRIPTION
**Pull request by mariamisa**

### CODE REVIEW

File: client/src/components/AddProduct/index.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              className="form-input"
+            />
+          </Form.Item>
+          <div className="cat-div">
+            <span>نوع اللوحة : </span>
+            <Select style={{ width: 120 }} onChange={handleCategories}>
+              <Option value="nature">طبيعة</Option>
+              <Option value="islamic">اسلامي</Option>
+              <Option value="sky">سماء</Option>
+              <Option value="hertage">ثقافة</Option>
```
Comment (Mu7ammadAbed) [inline client/src/components/AddProduct/index.js:117]:
```suggestion
              <Option value="hertiage">ثقافة</Option>
```

File: client/src/components/AddProduct/index.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            <Input
+              prefix={<MdDescription />}
+              placeholder="تفاصيل اللوحة"
+              className="form-input"
+            />
+          </Form.Item>
+          <div className="cat-div">
+            <span>نوع اللوحة : </span>
+            <Select style={{ width: 120 }} onChange={handleCategories}>
+              <Option value="nature">طبيعة</Option>
```
Comment (Mu7ammadAbed) [inline client/src/components/AddProduct/index.js:114]:
```suggestion
              <Option value="landscape">طبيعة</Option>
```

File: client/src/components/ProfileArtist/index.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+  const { artistId } = match.params;
+
+  const getArtistProfile = async (id) => {
+    try {
+      const { data: artistData } = await Axios.get(`/api/v1/profile/${id}`);
+      setProfileData(artistData.data[0]);
+    } catch (err) {
+      let e;
+      if (err.response.data.message === "Sorry There's no artist for this id") {
+        e = 'لا يوجد فنان لهذا الاي دي';
```
Comment (Mu7ammadAbed) [inline client/src/components/ProfileArtist/index.js:33]:
```suggestion
        e = 'لا يوجد فنان بهذا المعرف';
```

File: test/paintings.test.js [hunk 4]
```diff
@@ -99,73 +99,75 @@ describe('Delete painting )', () => {
   });
 });
 
-describe('POST /painting', () => {
-  const filePath = `${__dirname}/1610-Brain-Games-Study-workout.jpg`;
+// describe('POST /painting', () => {
```
Comment (Mu7ammadAbed) [inline test/paintings.test.js:102]: Why did you commented those tests ?

File: client/src/components/ProfileArtist/index.js [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+  const [loaded, setLoaded] = useState(false);
+  const [ArtistImg, setArtistImg] = useState();
+  const cloudinaryLink =
+    'https://res.cloudinary.com/dacf3uopo/image/upload/v1593353472/';
+  const { artistId } = match.params;
+
+  const getArtistProfile = async (id) => {
+    try {
+      const { data: artistData } = await Axios.get(`/api/v1/profile/${id}`);
+      setProfileData(artistData.data[0]);
```
Comment (ranasobeid95) [inline client/src/components/ProfileArtist/index.js:29]:
```suggestion
     const {
        data: { data: artistData },
      } = await Axios.get(`/api/v1/profile/${id}`);
      setProfileData(artistData[0]);
```

File: client/src/components/ProfileArtist/index.js [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+                  )
+                }
+              </AuthorizationContext.Consumer>
+              <p>{profileData.bio}</p>
+              <p>
+                <span>رقم الهاتف : </span> {profileData.mobile_no}
+              </p>
+              <div className="social_media">
+                {profileData.social_media_accounts[0] && (
+                  <a href={profileData.social_media_accounts[0]}>
```
Comment (ranasobeid95) [inline client/src/components/ProfileArtist/index.js:187]:
```suggestion
                {profileData.social_media_accounts && (
                <a
                    href={
                      profileData.social_media_accounts
                        ? profileData.social_media_accounts[0]
                        : null
                    }
                  >
```

**Pull request merged**